kuksa-sdk/src/test/kotlin/org/eclipse/kuksa/subscription/DataBrokerSubscriberTest.kt

```diff
@@ -78,19 +78,17 @@ class DataBrokerSubscriberTest : BehaviorSpec({
                         databrokerTransporter.updateRandomFloatValue(otherVssPath)
 
                         then("The Observer is notified about both changes") {
-                            verify(timeout = 100L) {
+                            verify(timeout = 100L, exactly = 3) {
```

Comment:
and here exactly=2?

Reply:
We removed the clearMocks and re-used the same listener, therefore after the previous subscribe (line 60) + the update from the previous test (line 63) + the update in this test (line 77) makes 3 exactly three updates :)